web/codegen.ts

```diff
@@ -0,0 +1,79 @@
+import { CodegenConfig } from "@graphql-codegen/cli";
+
+const rootGQLDirectory = "src/services/gql/";
+
+const rootGenerateDirectory = `${rootGQLDirectory}__gen__/`;
+
+const config: CodegenConfig = {
+  overwrite: true,
+  schema: "../server/gql/*.graphql",
+  documents: [`${rootGQLDirectory}fragments/*.ts`, `${rootGQLDirectory}queries/*.ts`],
+  generates: {
+    [rootGenerateDirectory]: {
```

Comment:
we don't need plugins any more?
["typescript", "typescript-operations", "typescript-react-apollo"],

Reply:
True! Removed [here](https://github.com/reearth/reearth/pull/536/commits/320e2b4611444a535d9bbc3ec8e3f6d25618817a)